rotkehlchen/db/snapshots.py

```diff
@@ -110,11 +110,11 @@ def _csv_to_dict(file: Path) -> List[Dict[str, str]]:
             return list(csv_reader)
 
     def create_zip(
-        self,
-        timed_balances: List[DBAssetBalance],
-        timed_location_data: List[LocationData],
-        main_currency: Asset,
-        main_currency_price: Price,
+            self,
+            timed_balances: List[DBAssetBalance],
+            timed_location_data: List[LocationData],
+            main_currency: Asset,
+            main_currency_price: Price,
```

Comment:
mmmm I though that we were using 4 here but seems that this is indeed what we are respecting